packages/terra-collapsible-menu-view/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Changed
+  * Updated Jest snapshot due to `terra-menu` changes
+  * Updated WDIO screenshots due to `terra-menu` changes
+
```

Comment:
This is no longer needed with the recent `dangerfile.js` updates.

Reply:
reverted in https://github.com/cerner/terra-framework/pull/1626/commits/131a931bb445c9aeab73a4e5174592ed7d503b99